godtools/App/Services/Renderer/Views/MobileContent/Views/ContentLink/MobileContentLinkViewModel.swift

```diff
@@ -7,25 +7,26 @@
 //
 
 import UIKit
+import GodToolsToolParser
 
 class MobileContentLinkViewModel: MobileContentLinkViewModelType {
     
-    private let linkModel: ContentLinkModelType
-    private let rendererPageModel: MobileContentRendererPageModel
+    private let linkModel: Link
+    private let renderedPageContext: MobileContentRenderedPageContext
     private let mobileContentAnalytics: MobileContentAnalytics
     private let fontService: FontService
     private let fontSize: CGFloat = 18
     private let fontWeight: UIFont.Weight = .regular
     
     let titleColor: UIColor
     
-    required init(linkModel: ContentLinkModelType, rendererPageModel: MobileContentRendererPageModel, mobileContentAnalytics: MobileContentAnalytics, fontService: FontService) {
+    required init(linkModel: Link, renderedPageContext: MobileContentRenderedPageContext, mobileContentAnalytics: MobileContentAnalytics, fontService: FontService) {
         
         self.linkModel = linkModel
-        self.rendererPageModel = rendererPageModel
+        self.renderedPageContext = renderedPageContext
         self.mobileContentAnalytics = mobileContentAnalytics
         self.fontService = fontService
-        self.titleColor = linkModel.getTextColor()?.uiColor ?? rendererPageModel.pageColors.primaryColor.uiColor
+        self.titleColor = linkModel.textColor
```

Comment:
I think this suffers from the same text styles issues that buttons suffered from. `Link` probably currently implements `Styles` which is what is giving you the `textColor` property, but that doesn't take a `text-color` set on the text node into account.

Reply:
I'm going to work up another PR to do the same thing I did to Buttons to Links

Reply:
https://github.com/CruGlobal/kotlin-mpp-godtools-tool-parser/pull/230

Reply:
Hey @frett will I need to update this to use the ```linkModel.text.textColor```?  Or does your change make ```linkMode.textColor``` equivalent to that?

Reply:
you will need to change to `linkModel.text.textColor`.

`linkModel.textColor` will actually become invalid when you upgrade past 0.5.0 since that property is no longer present on the `Link` model itself